docs/ref/utils.txt

```diff
@@ -867,6 +867,10 @@ appropriate entities.
         >>> slugify('你好 World', allow_unicode=True)
         '你好-world'
 
+    .. versionchanged:: 3.2
+
+        This function now also removes trailing underscores and dashes.
```

Comment:
`In older versions, leading and trailing underscores and dashes weren't removed.`